Templates/DefaultProject/Template/ExportScripts/export_standalone_monolithic_engine_centric.py

```diff
@@ -0,0 +1,161 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+
+import argparse
+import glob
+import logging
+import os
+import o3de.export_project as exp
+import pathlib
+import sys
+import time
+
+from o3de.export_project import process_command
+from o3de import manifest
+
+project_json_data = manifest.get_project_json_data(project_path=o3de_context.project_path)
```

Comment:
Can we move most of the code into a function and have a `if __name__ == "__main__"` condition in this script to call the function? (Or even better yet, isolate these steps into more separate concrete actions and call them in sequence).
This pattern is done to prevent this logic from executing if someone just imports this script as a module.

Reply:
So it turns out `if __name__ == "__main__"` will not work because the export script is run by the export_project.py script. As such it is not `__main__`.

We can get an equivalent gating mechanism using `if "o3de_context" in globals()`, which provides the extra benefit that we know only O3DE export related code will run in that section.

Reply:
I see that now
```
utils.load_and_execute_script(target_script_path, o3de_context = o3de_context, o3de_logger=logging.getLogger())
```